neo21/src/main/java/com/envyful/gts/forge/ui/EditItemDurationUI.java

```diff
@@ -55,6 +55,7 @@ public static void openUI(ForgeEnvyPlayer player, double time, boolean error) {
                             }
 
                             UtilPlayer.runCommand(player.getParent(), "gts sell " + itemInHand.getCount() + " " + time + " " + submitted.getInput());
+                            PlatformProxy.runSync(player.getParent()::closeContainer);
```

Comment:
Same again here, clicking the button should be closing the UI so why is this here?